includes/woop.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Woo on Plans.
+ *
+ * @since 1.8.1
+ * @package WC_Calypso_Bridge
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Init woop.
+ *
+ * @return void
+ */
+function wc_calypso_bridge_woop_init() {
+	// Don't load mods in cronjobs.
+	if ( defined( 'DOING_CRON' ) ) {
+		return;
+	}
+
+	if ( ! class_exists( 'Atomic_Persistent_Data' ) ) {
+		return;
+	}
+
+	// Gate Woo on plans behind the site sticker "woop" feature flag.
+	if ( ! ( new Atomic_Persistent_Data() )->site_sticker_woop ) {
```

Comment:
> Would the class use anything from woocommerce?

No, its just reading from a file.

> Should this be before woocommerce check?

Eventually this will be removed and the woop stuff will apply to all sites, I suspected the file lookup was faster than the get_option lookup but its probably over thinking it either way.